CHANGELOG.md

```diff
@@ -6,7 +6,27 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.2.0 (2021-04-13)
+
+This release adds major new features since the 2.1.1 release.
+We deem it moderate priority for upgrading.
+
+This release adds the skip-scan optimization, which significantly 
+improves performance of queries with DISTINCT ON. This optimization is 
+not available for queries on distributed hypertables.
+
+For multinode, this release adds a function to create a distributed 
+restore point, which allows the consistent restore of a multinode 
+cluster from a backup. The release also includes improvements to 
+distributed query performance and memory usage.
```

Comment:
Not a biggie, but the performance and memory usage stuff is immediately repeated below in more detail.